app/controllers/guide_controller.rb

```diff
@@ -1,11 +1,11 @@
 class GuideController < ApplicationController
   include ApiRedirectable
-  include Previewable
   include Cacheable
```

Comment:
Quick thought re including the `Cacheable` module. The main thing it does is `set_expiry` unless there's an edition param present. That's not really relevant for the migrated formats anymore. I guess we could leave it in the controllers for now (since it doesn't do any harm) and just make sure we remove it once we've migrated all formats and we're not sending user from Publisher to private-frontend anymore.